draft-thomassen-dnsop-multialgo.md

```diff
@@ -86,13 +86,21 @@ via its trust anchors. This poses a problem for (at least) the following cases:
     and SSR costs, it seems desirable to find a way for publishing the new trust
     anchor without introducing the new algorithm into the zone just yet.
 
+<!-- Viktor: What is an "SSR cost"? -->
```

Comment:
Security, Stability, Resilience. Might be ICANN speak. Replaced by "operational risk".